src/assets/styles/components/spinner.scss

```diff
@@ -6,14 +6,14 @@
   height: $spinner-lg;
   margin-top: calc(-#{$spinner-lg} / 1.5);
   margin-left: calc(-#{$spinner-lg} / 2);
-  background-image: url(./assets/images/spinner.png);
+  background-image: url(../../images/spinner.png);
```

Comment:
Just curious here, did the way Angular load assets in version 11 change or something? Why the need to update a bunch of references in scss?

Reply:
No, I don't believe that this was due to a local change. I started with a clean branch.